packages/blocks/google-cloud/src/lib/actions/get-recommendations-cli-action.ts

```diff
@@ -0,0 +1,296 @@
+import { createAction, Property } from '@openops/blocks-framework';
+import {
+  getUseHostSessionProperty,
+  googleCloudAuth,
+  handleCliError,
+  tryParseJson,
+} from '@openops/common';
+import { runCommand, runCommands } from '../google-cloud-cli';
+
+export const getRecommendationsAction = createAction({
+  auth: googleCloudAuth,
+  name: 'google_get_recommendations_cli',
+  description: 'Fetch the recommendations for the provided recommenders',
```

Comment:
nitpicky:
maybe "for the selected"
provided sounds like it's input from another step